azext_iot/tests/central/json/property_validation_template.json

```diff
@@ -7,8 +7,129 @@
+              "@type": "Property",
+              "displayName": "testComponent",
+              "name": "testComponent",
+              "schema": "boolean",
+              "writable": true
+            },
+            {
+              "@id": "dtmi:sampleApp:_rpgcmdpo:component1PropReadonly;1",
+              "@type": "Property",
+              "displayName": "component1PropReadonly",
+              "name": "component1PropReadonly",
+              "schema": "boolean",
+              "writable": false
+            },
+            {
+              "@id": "dtmi:sampleApp:_rpgcmdpo:component1Prop2;1",
+              "@type": "Property",
+              "displayName": "component1Prop2",
+              "name": "component1Prop2",
+              "schema": "boolean",
+              "writable": true
+            }
+          ],
+          "displayName": "Component"
+        }
+      },
+      {
+        "@id": "urn:rigado:RS40_Occupancy_Sensor:RS40OccupancySensorV36fy:3",
+        "@type": "Component",
+        "displayName": "Component",
+        "name": "RS40OccupancySensorV36fy",
+        "schema": {
+          "@id": "dtmi:cliIntegrationtestApp:RS40OccupancySensorV36fy;4",
+          "@type": "Interface",
+          "contents": [
+            {
+              "@id": "dtmi:cliIntegrationtestApp:RS40OccupancySensorV36fy:component2prop;1",
+              "@type": "Property",
+              "displayName": "component2prop",
+              "name": "component2prop",
+              "schema": "boolean",
+              "writable": true
+            },
+            {
+              "@id": "dtmi:cliIntegrationtestApp:RS40OccupancySensorV36fy:testComponent;2",
+              "@type": "Property",
+              "displayName": "testComponent",
+              "name": "testComponent",
+              "schema": "boolean",
+              "writable": true
+            },
+            {
+              "@id": "dtmi:cliIntegrationtestApp:RS40OccupancySensorV36fy:component2PropReadonly;3",
+              "@type": "Property",
+              "displayName": "component2PropReadonly",
+              "name": "component2PropReadonly",
+              "schema": "boolean",
+              "writable": false
+            },
+            {
+              "@id": "dtmi:cliIntegrationtestApp:RS40OccupancySensorV36fy:component2Prop2;4",
+              "@type": "Property",
+              "displayName": "component2Prop2",
+              "name": "component2Prop2",
+              "schema": "boolean",
+              "writable": true
+            },
+            {
+              "@id": "dtmi:cliIntegrationtestApp:RS40OccupancySensorV36fy:component1Telemetry;5",
+              "@type": "Telemetry",
+              "displayName": "component1Telemetry",
+              "name": "component1Telemetry",
+              "schema": "double"
+            }
+          ],
+          "displayName": "Component"
+        }
+      },
+      {
+        "@id": "urn:rigado:RS40_Occupancy_Sensor:addRootPropertyReadOnly:5",
```

Comment:
nit - why are some of these URN's and others DTMI's?

Reply:
If central supports only v2, then it should always be dtmi.

Reply:
Wont  fix - dtdlv1 and v2 are supported today by public API and will continue to do so till end of june.-